vue-frontend/src/components/portfolio/BrandingSection.vue

```diff
@@ -2,7 +2,7 @@
   <section class="tw-bg-white">
     <div
       v-if="response.solution"
-      class="tw-container tw-py-40 lg:tw-py-80 tw-relative"
+      class="tw-container tw-py-14 sm:tw-py-20 lg:tw-py-80 tw-relative"
```

Comment:
`tw-py-14 sm:tw-py-20 lg:tw-py-80`
Which will be applicable?

Reply:
All.

Reply:
From small to large, `tw-py-20` will be applicable, and from lg to rest `tw-py-80` will be applicable, what is use of `tw-py-14`?